doc/dev/developer_guide/tests-windows.rst

```diff
@@ -0,0 +1,126 @@
+.. _dev-testing-windows:
+
+=================
+Testing - Windows
+=================
+
+Since Pacific, the Ceph client tools and libraries can be natively used on
+Windows. This allows Windows nodes to consume Ceph without additional layers
+such as iSCSI gateways or SMB shares.
+
+A significant amount of unit tests and integration tests were ported in order
+to ensure that these components continue to function properly on Windows.
+
+Windows CI Job
+==============
+
+The `Windows CI job`_ performs the following steps for each GitHub pull request:
+
+* spin up a Linux VM in which to build the server-side (Linux) Ceph binaries
+  and cross-compile the Windows (client) binaries.
+* recreate the Linux VM and start a Ceph vstart cluster
+* boot a Windows VM and run the Ceph tests there
+
+`A small PowerShell framework`_ parallelizes the tests, aggregates the results
+and isolates or skips certain tests that are known to be flaky.
+
+The console output can contain compilation errors as well as the name of the
+tests that failed. To get the console output of the failing tests as well as
+Ceph and operating system logs, please check the build artifacts from the
+Jenkins "Status" page.
+
+.. image:: ../../images/windows_ci_status_page.png
+      :align: center
+
+The Windows CI artifacts can be downloaded as a zip archive or viwed inside
+the browser. Click the "artifacts" button to see the contents of the artifacts
+folder.
+
+.. image:: ../../images/windows_ci_artifacts.png
+      :align: center
+
+Artiact contents:
```

Comment:
Artifact

Reply:
Thanks. Sorry for the typos, wrote this in a hurry.

Reply:
No, no. These are notes to self. I'll correct all this stuff in a future PR.

I think this is a great contribution. I'm not meaning to be nit-picky.

Also, you need to add a Signed-off-by: line.

Reply:
Signed-off-by: line instructions can be read here:

https://docs.ceph.com/en/reef/dev/developer_guide/basic-workflow/#configuring-your-local-environment

Reply:
Actually, you just need to put "doc:" at the beginning of the title of this PR.